Documentation/Internal/ContainerLogV2-Linux.xml

```diff
@@ -25,6 +25,8 @@
       <Column name="Computer" type="str" mdstype="mt:wstr" />
       <Column name="ContainerName" type="str" mdstype="mt:wstr" />
       <Column name="AzureResourceId" type="str" mdstype="mt:wstr" />
+      <Column name="KubernetesMetadata" type="str" mdstype="mt:wstr" />
+      <Column name="LogLevel" type="str" mdstype="mt:wstr" />
```

Comment:
LogLevel is not supported in geneva and this can be removed.

Reply:
changed

Reply:
removed